some.json

```diff
@@ -0,0 +1,13 @@
+{
+  "email": "",
+  "prBoard": "",
+  "comment": {
+    "title": "",
+    "description": "",
+    "board": "",
+    "type": "",
+    "priority": "",
+    "assignee": "",
+    "status": ""
```

Comment:
/ticket what is love? -d baby don't hurt me

Reply:
Here's your ticket: [ what is love?](https://ticketgen19.atlassian.net/browse/IKKI-40)